src/TestHelper/RequestObjectTestHelper.php

```diff
@@ -144,14 +144,16 @@ public function getRequestProperties(RequestObjectInterface $object): array
         $methodsToCheck = \array_filter(
             \array_diff($instanceMethods, $interfaceMethods),
             static function (string $method): bool {
-                return \strncmp($method, 'get', 3) === 0;
+                return \strncmp($method, 'get', 3) === 0 ||
+                    \strncmp($method, 'is', 2) === 0;
             }
         );
 
         $actual = [];
-        foreach ($methodsToCheck as $method) {
-            $property = \lcfirst(\substr($method, 3));
+        \asort($methodsToCheck);
 
+        foreach ($methodsToCheck as $method) {
+            $property = \lcfirst(\substr($method, \strncmp($method, 'get', 3) === 0 ? 3 : 2));
```

Comment:
difficult to understand what is happening between brackets. Can this be broken please.

Reply:
Done!